examples/cursor_sync_custom_overlay.py

```diff
@@ -0,0 +1,106 @@
+"""
+Cursor sync via custom overlay
+==============================
+
+Create 2 seperate viewers and mirror the position of the cursor between the
+parent and the child via a vispy overlay.
+
+Read up on the view model here: https://napari.org/stable/guides/napari_models.html
```

Comment:
I don't actually know the answer to this: can we use sphinx/myst links here rather than html links? Then this should be (I think):

```suggestion
Read up on the view model here: {ref}`napari-model-event`
```

CC @melissawm 

This is indeed a good idea because it seems [the page has been moved in dev](https://github.com/napari/docs/blob/1d5c5b956ce00ac5d942c0d1d0e2dc2edf63a1dd/docs/developers/architecture/napari_models.md). 😅

Reply:
That's correct 👍🏻

Reply:
Done